swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
```

Comment:
This works! Though to take advantage of the benefits of inheritance, you can also call the parent class's constructor:
`super().__init__(condition, category='Clothing', age)`